include/model/GPRModel.hpp

```diff
@@ -36,80 +36,38 @@
 #include "model/Model.hpp"
 #include "model/ModelFramework.hpp"
 #include "model/ScikitLearnFramework.hpp"
-#include "statespace/StateSpace.hpp"
 
 namespace libcozmo {
 namespace model {
 
+/// This class implements a Gaussian Process Regressor(GPR), a type of
+/// multi-dimensional linear regression model, from Scikit-Learn.
```

Comment:
remove `, from Scikit-Learn.` because technically it could be trained in a different framework